v2/features/connection/networks.ts

```diff
@@ -0,0 +1,45 @@
+import arbitrum from "public/arbitrum.svg";
+import aurora from "public/aurora.svg";
+import ethereum from "public/ethereum.svg";
+import oec from "public/oec.svg";
+import matic from "public/matic.svg";
+import moonriver from "public/moonriver.svg";
+
+type Network = {
+  name: string;
+  icon: any;
+  chainId: number;
+};
+
+export const networks: Network[] = [
+  {
```

Comment:
I can make pf-core include a visibleName field for the chains. If you could make the icon equal to `"public/${chainName}.svg" `(if it exists, or a default icon otherwise), we could remove this entire structure and never need to update it. WDYT?

Reply:
We can do that but with the icons it's a bit more complicated. Doing a static import lets you preload assets with priority on the frontend. You'd also have to make an assumption about file format. Overall, I think this isn't much of a concern if we add a new chain once a month or so. Having said that, having display name of the chain in pf-core will be useful.